bundle-workflow/infra/bin/distribution-build.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from '@aws-cdk/core';
+import { DistributionBuildStack, DistributionBuildStackProps } from '../lib/distribution-build-stack';
+
+// arn:aws:iam::336984078605:role/JenkinsStackDev-MainNodeInstanceRole6FA3C5C9-1PQFWOWDYN42U
+
+const stacks: DistributionBuildStackProps[] = [{
+  name: 'DistributionBuildStack-Staging',
+  bucketName: 'teststatus-buildbucket-1otmnpvrkhfji',
+  buildRoleName: 'opensearch-build',
+  bundleRoleName: 'opensearch-bundle',
+  testRoleName: 'opensearch-test',
+  env: {
+    account: '791023489099',
```

Comment:
99% sure we can include the account numbers in PR, but I need to confirm this.

Reply:
FYI - We should not, using our internal repository to store values - converting these over to parameters